emails/views.py

```diff
@@ -336,11 +341,41 @@ def _sns_message(message_json):
     return ses_relay_email(from_address, relay_address, subject, message_body)
 
 
+def _get_attachment_metrics(part):
+    incr_if_enabled('email_with_attachment', 1)
+    fn = part.get_filename()
+    if fn:
+        extension = os.path.splitext(fn)[1]
+    else:
+        extension = mimetypes.guess_extension(
+            part.get_content_type()
+        )
+    payload = part.get_payload(decode=True)
+    logger.error(
+        'Attachment found in email',
+        extra={
+            'content-type': part.get_content_type(),
+            'extension': extension,
+            'payload-size': len(payload)
+        }
+    )
+    tag_type = 'attachment'
+    attachment_extension_tag = generate_tag(tag_type, part.get_content_type())
+    attachment_content_type_tag = generate_tag(tag_type, extension)
+    histogram_if_enabled(
+        'attachment.size',
+        len(payload),
+        [attachment_extension_tag, attachment_content_type_tag]
+    )
+    return part.get_content_type(), extension, len(payload)
+
+
 def _get_text_and_html_content(email_message):
     text_content = None
     html_content = None
     has_attachment = False
     if email_message.is_multipart():
+        email_count = 0
```

Comment:
Should this be more explicit name since it's only incremented for attachments? `email_attachment_count` ? `attachments_in_email` ?